tools/docker/docker-compose.yml

```diff
@@ -8,11 +8,20 @@ services:
       dockerfile: tools/docker/Dockerfile
     ports:
       - "8080:8080"
-    volumes:
-      - sqlite_db:/opt/app-root/src/sqlite
     environment:
       - AE_HOST=0.0.0.0
-      - AE_DATABASE_URL=sqlite:///./sqlite/test.db
```

Comment:
If sqlite will be remove, also you could update this line: 
https://github.com/benthomasson/ansible-events-ui/blob/main/tools/docker/Dockerfile#L19